### PR TITLE
Adding option to create sandboxes from existing config

### PR DESCRIPTION
**Pull request by pchand-amplify**
To create a sandbox from config : 
`$> asiaq sandbox oreo`

To create another sandbox from `oreo` config (without creating a temp git branch or symlink):
`$> asiaq sandbox oreo2 --config-dir oreo`

### CODE REVIEW

**Comment by pchand-amplify** [comment]
@aoskotsky-amplify : any comments?

File: disco_aws_automation/asiaq_cli.py [hunk 1]
```diff
@@ -100,7 +106,7 @@ def _update_s3_configs(self, sandbox_name):
         s3_bucket = boto3.resource("s3").Bucket(name=bucket_name)
         for sync_line in config_sync_option.split("\n"):
             local_name, remote_dir = sync_line.split()
-            local_config_path = normalize_path("sandboxes", sandbox_name, local_name)
+            local_config_path = normalize_path("sandboxes", config_dir, local_name)
```
Comment (aoskotsky-amplify) [inline disco_aws_automation/asiaq_cli.py]: Does this mean that `config_dir` is a required arg to the command?

Comment (pchand-amplify) [reply disco_aws_automation/asiaq_cli.py]: It is optional.

Comment (aoskotsky-amplify) [reply disco_aws_automation/asiaq_cli.py]: Oh I see

**Pull request merged**